spec/models/validators/utaac_decision_validator_spec.rb

```diff
@@ -10,16 +10,14 @@
       title: double,
       summary: double,
       body: "body",
-      tribunal_decision_category: category,
+      tribunal_decision_categories: [double],
       tribunal_decision_decision_date: "2015-11-01",
       tribunal_decision_judges: [double],
-      tribunal_decision_sub_category: sub_category,
+      tribunal_decision_sub_categories: [double],
     )
   }
   let(:document_type) { "utaac_decision" }
 
   subject(:validatable) { UtaacDecisionValidator.new(entity) }
 
-  it_behaves_like "tribunal decision sub_category validator"
```

Comment:
Removing this shared example means that this spec now has no examples in it (putting it in the same boat as `EmploymentAppealTribunalDecisionValidator`).  As these two validators both have a `tribunal_decision_sub_categories` field, perhaps we could copy and repurpose the "tribunal decision sub_category validator " shared examples to provide some coverage?